[0m[[0minfo[0m] [0m[32mListsSuite:[0m[0m
[0m[[0minfo[0m] [0m[32m- one plus one is two[0m[0m
[0m[[0minfo[0m] [0m[31m- one plus one is three? *** FAILED ***[0m[0m
[0m[[0minfo[0m] [0m[31m  org.scalatest.exceptions.TestFailedException was thrown. (ListsSuite.scala:50)[0m[0m
[0m[[0minfo[0m] [0m[31m- details why one plus one is not three *** FAILED ***[0m[0m
[0m[[0minfo[0m] [0m[31m  2 did not equal 3 (ListsSuite.scala:75)[0m[0m
[0m[[0minfo[0m] [0m[32m- intNotZero throws an exception if its argument is 0[0m[0m
[0m[[0minfo[0m] [0m[32m- sum of a few numbers[0m[0m
[0m[[0minfo[0m] [0m[32m- sum of zeros list[0m[0m
[0m[[0minfo[0m] [0m[32m- sum of negative numbers[0m[0m
[0m[[0minfo[0m] [0m[32m- sum applyed on empty list[0m[0m
[0m[[0minfo[0m] [0m[32m- sum in list with repeated elements[0m[0m
[0m[[0minfo[0m] [0m[32m- max of a few numbers[0m[0m
[0m[[0minfo[0m] [0m[32m- max of zeros list[0m[0m
[0m[[0minfo[0m] [0m[32m- max of negative numbers[0m[0m
[0m[[0minfo[0m] [0m[32m- max throws an exception if its argument is an empty list[0m[0m
[0m[[0minfo[0m] [0m[32m- max in list with repeated elements[0m[0m
